src/image/p5.Image.js

```diff
@@ -828,4 +831,87 @@ p5.Image.prototype.save = function(filename, extension) {
   p5.prototype.saveCanvas(this.canvas, filename, extension);
 };
 
+p5.Image.prototype.play = function() {
+  if (this.gifProperties === null || this.gifProperties.playing === true) {
+    return;
+  } else {
+    this.gifProperties.playing = true;
+  }
+};
+
+p5.Image.prototype.pause = function() {
+  if (this.gifProperties === null || this.gifProperties.playing === false) {
+    return;
+  } else {
+    this.gifProperties.playing = false;
+  }
+};
+
+p5.Image.prototype.delay = function(d) {
+  if (this.gifProperties === null) {
+    console.error('This is not a gif');
```

Comment:
we generally try to avoid `console.error` to allow things to fail more gracefully. `console.log` should be fine

Reply:
Most of these are outdated I think but yes console.error got it! I was going to just route this thru friendly error system for now.